src/index.ts

```diff
@@ -9,7 +9,13 @@ import { run } from "./run";
  * Run the plugin as a GitHub Action instance.
  */
 async function actionRun() {
-  const env = Value.Decode(envSchema, process.env);
+  const payloadEnv = {
+    SUPABASE_KEY: process.env.SUPABASE_KEY,
+    SUPABASE_URL: process.env.SUPABASE_URL,
+    UBIQUIBOT_PUBLIC_KEY: process.env.UBIQUIBOT_PUBLIC_KEY || "temporarily-disabled",
```

Comment:
Do we want a default here, shouldn't we display an error and stop the run?

Reply:
We should definitely throw an error.

Reply:
https://github.com/ubiquity-os-marketplace/daemon-pricing/blob/c9c53ba0d12314943e206100cee8321f9c4adc57/src/worker.ts#L32

It's currently not used in this codebase. I wasn't sure whether it was defined or not but like where it's used I thought it should be explicitly stated that it's disabled.

Unsure what to do considering this, please lmk

Reply:
Ha yes I remember this part now. It can be safely ignored then.

Reply:
Let's delete we deprecated the term UbiquiBot officially a couple days ago.